doc/release-notes.md

```diff
@@ -1 +1,215 @@
-See https://github.com/bitcoin-core/bitcoin-devwiki/wiki/27.0-Release-Notes-Draft
+Bitcoin Core version 27.0 is now available from:
+
+  <https://bitcoincore.org/bin/bitcoin-core-27.0/>
+
+This release includes new features, various bug fixes and performance
+improvements, as well as updated translations.
+
+Please report bugs using the issue tracker at GitHub:
+
+  <https://github.com/bitcoin/bitcoin/issues>
+
+To receive security and update notifications, please subscribe to:
+
+  <https://bitcoincore.org/en/list/announcements/join/>
+
+How to Upgrade
+==============
+
+If you are running an older version, shut it down. Wait until it has completely
+shut down (which might take a few minutes in some cases), then run the
+installer (on Windows) or just copy over `/Applications/Bitcoin-Qt` (on macOS)
+or `bitcoind`/`bitcoin-qt` (on Linux).
+
+Upgrading directly from a version of Bitcoin Core that has reached its EOL is
+possible, but it might take some time if the data directory needs to be migrated. Old
+wallet versions of Bitcoin Core are generally supported.
+
+Compatibility
+==============
+
+Bitcoin Core is supported and extensively tested on operating systems
+using the Linux Kernel 3.17+, macOS 11.0+, and Windows 7 and newer. Bitcoin
+Core should also work on most other Unix-like systems but is not as
+frequently tested on them. It is not recommended to use Bitcoin Core on
+unsupported systems.
+
+Notable changes
+===============
+
+libbitcoinconsensus
+-------------------
+
+- libbitcoinconsensus is deprecated and will be removed for v28. This library has
+  existed for nearly 10 years with very little known uptake or impact. It has
+  become a maintenance burden.
+
+  The underlying functionality does not change between versions, so any users of
+  the library can continue to use the final release indefinitely, with the
+  understanding that Taproot is its final consensus update.
+
+  In the future, libbitcoinkernel will provide a much more useful API that is
+  aware of the UTXO set, and therefore be able to fully validate transactions and
+  blocks. (#29189)
+
+mempool.dat compatibility
+-------------------------
+
+- The `mempool.dat` file created by -persistmempool or the savemempool RPC will
+  be written in a new format, which can not be read by previous software
+  releases. To allow for a downgrade, a temporary setting `-persistmempoolv1` has
+  been added to fall back to the legacy format. This new format includes the XOR'ing
```

Comment:
"This new format" right after talking about the legacy format is consusing imo.

Suggested rewrite 1)first explains the new format, 2) then describes how to fall back, 3) adds the PR number:

```
- The `mempool.dat` file created by -persistmempool or the savemempool RPC will
  be written in a new format. This new format includes the XOR'ing of transaction
  contents to mitigate issues where external programs (such as anti-virus) attempt
  to interpret and potentially modify the file.

  This new format can not be read by previous software releases. To allow for a
  downgrade, a temporary setting `-persistmempoolv1` has been added to fall back
  to the legacy format. (#28207)
```